### PR TITLE
Support large font for session tag and timetable tab

### PR DESCRIPTION
**Pull request by Aniokrait**
## Issue
- sorry, I haven't opened any issue. Please tell me if I should open it before PR and I'll create the one.
- I've found the problem that the session tags and the timetable tabs don't support large font size.
  - Please refer to the screenshot I attached bottom to see exactly what happend.
### how to reproduce: 
- Device: Pixel6 emulator
- Change the font size to the largest in the device settings.
- Open the app.

## Overview (Required)
- Stopped specifying a fixed height and instead set a minimum height.

### large font size
Before | After
:--: | :--:
 | 
### normal font size
There is no difference.
Before | After
:--: | :--:
 |

### CODE REVIEW

**Comment by takahirom** [comment]
Could you add a screenshot test so that you can confirm the large font?

https://github.com/DroidKaigi/conference-app-2023/blob/3395eaf2e5212ca236114a9942a700dfc6bb483d/feature/sessions/src/test/java/io/github/droidkaigi/confsched2023/sessions/TimetableItemDetailScreenTest.kt#L120

You can add it here

https://github.com/DroidKaigi/conference-app-2023/blob/250b7bb2d4b4b8aafaeffe45bc0dc3934d225e91/feature/sessions/src/test/java/io/github/droidkaigi/confsched2023/sessions/TimetableScreenTest.kt#L41

**Comment by Aniokrait** [comment]
Sure! I'll add the tests.

**Comment by Aniokrait** [comment]
Thanks to your precice instructions, I could write tests easily!
By the way, the original test case names looked like not matching the naming convention. It might be none of my business, but I also corrected those names.

**Review by takahirom — APPROVED** [review]
Looks great!

File: feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/component/SessionTag.kt [hunk 1]
```diff
@@ -29,7 +30,7 @@ fun SessionTag(
 ) {
     Box(
         modifier = modifier
-            .height(24.dp)
+            .defaultMinSize(minHeight = 24.dp)
```
Comment (takahirom) [inline feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/component/SessionTag.kt:33]: 👍

**Pull request merged**